core/services/chainlink/testdata/config-full.toml

```
@@ -0,0 +1,286 @@
+Dev = true
+ExplorerURL = 'http://explorer.url'
+InsecureFastScrypt = true
+ReaperExpiration = '168h0m0s'
+Root = 'test/root/dir'
+ShutdownGracePeriod = '10s'
+FeatureFeedsManager = true
+FeatureUICSAKeys = true
+FeatureLogPoller = true
+FMDefaultTransactionQueueDepth = 100
+FMSimulateTransactions = true
+FeatureOffchainReporting2 = true
+FeatureOffchainReporting = true
```

Should we group feature flags up in a table?
```suggestion
FMDefaultTransactionQueueDepth = 100
FMSimulateTransactions = true

[Feature]
FeedsManager = true
UICSAKeys = true
LogPoller = true
OffchainReporting2 = true
OffchainReporting = true
```
Alternatively, the ones with their own tables (`[OCR]`, `[OCR2]`) could be implicit based on the presence of the table.